README.md

```diff
@@ -1,4 +1,4 @@
-# Ramme <img src="static/icon.png" width="70">
+# Ramme(Stephen.Kingsley process) <img src="static/icon.png" width="70">
```

Comment:
I welcome new ideas and changes! It's the power of open source 👍 
When you create pull-request please create one pull-request pr. feature 😄